pkg/tempo/automaticloggingprocessor/automaticloggingprocessor.go

```diff
@@ -191,7 +217,7 @@ func (p *automaticLoggingProcessor) spanKeyVals(span pdata.Span) []interface{} {
 	atts = append(atts, spanDuration(span))
 
 	atts = append(atts, p.cfg.Overrides.StatusKey)
-	atts = append(atts, span.Status().Code())
+	atts = append(atts, span.Status().Code().String())
```

Comment:
to reduce spam i wonder if we should only log this if it does not equal STATUS_CODE_UNSET

Reply:
Yeah, good point. I don't have a strong opinion on this, I'm fine removing it.